pkgs/development/compilers/xa/xa.nix

```diff
@@ -15,13 +15,12 @@ stdenv.mkDerivation rec {
   dontConfigure = true;
 
   postPatch = ''
-    substitueInPlace \
+    substituteInPlace Makefile \
```

Comment:
Why to change the Makefile positioning as argument here?

Reply:
personal preference.

Reply:
I didn't know about it. The function always worked when I put the file as the last arg...